discovery-provider/src/queries/get_health.py

```diff
@@ -225,6 +206,8 @@ def get_health(args: GetHealthArgs, use_redis_cache: bool = True) -> Tuple[Dict,
     # fetch latest db state if:
     # we explicitly don't want to use redis cache or
     # value from redis cache is None
+    latest_indexed_block_num, latest_indexed_block_hash = get_latest_indexed_chain_block_from_redis(redis)
```

Comment:
can we move this to below get_latest_chain_block_from_redis? thats where it was originally

Reply:
I moved this fn here bc it wasn't used until lines 213-214 and would like to keep variable initializations and usages close together. can move if you feel strongly

Reply:
the bit that confused me was the placement relative to the comment. like the comments above say fetch latest db state if and then it gets the value from redis

Reply:
ill move the comment below this fn call as it was done originally